src/plugins/credential_management/opensearch_dashboards.json

```diff
@@ -2,6 +2,7 @@
   "id": "credentialManagement",
   "version": "1.0.0",
   "opensearchDashboardsVersion": "opensearchDashboards",
+  "configPath": ["credential_management"],
```

Comment:
you don't need to specify it as the config path by default uses the plugin id

Reply:
It's on purpose - 
Since id `credentialManagement` is different from folder name `credential_management`, it would output Error during yarn start without this configPath setup

```
 FATAL  Error: Unknown configuration key(s): "credentialManagement.materialPath". Check for spelling errors and ensure that expected plugins are installed.
```